spec/models/comment_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] I suggest that any custom methods in the models should have a corresponding unit test. So make sure that you add a test for the method `update_post_comments_counter` in your Comment model. :blush: